storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -11,10 +11,4 @@ In-tree storage drivers that are traditionally shipped with {product-title} are
 include::modules/persistent-storage-csi-migration-overview.adoc[leveloffset=+1]
 include::modules/persistent-storage-csi-migration-sc.adoc[leveloffset=+1]
 
-To change the default storage class, see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#change-default-storage-class_persistent-storage-csi-sc-manage[Changing the default storage class].
-
-include::modules/persistent-storage-csi-migration-vsphere.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-* xref:../../updating/updating_a_cluster/eus-eus-update.adoc#eus-eus-update[Performing an EUS-to-EUS update]
\ No newline at end of file
+To change the default storage class, see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#change-default-storage-class_persistent-storage-csi-sc-manage[Changing the default storage class].
```

Comment:
We're [not supposed to use intro text ](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)in additional references (even if it is helpful).
```suggestion
* xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#change-default-storage-class_persistent-storage-csi-sc-manage[Changing the default storage class].
```

Reply:
Got it. Thanks!